tests/foreman/destructive/test_remoteexecution.py

```diff
@@ -0,0 +1,125 @@
+"""Test for Remote Execution
+
+:Requirement: Remoteexecution
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: RemoteExecution
+
+:Assignee: pondrejk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from nailgun import client
+from nailgun.entity_mixins import TaskFailedError
+
+from robottelo.cli.factory import make_job_invocation
+from robottelo.cli.job_invocation import JobInvocation
+from robottelo.cli.task import Task
+
+CAPSULE_TARGET_VERSION = '6.10.z'
+
+pytestmark = pytest.mark.destructive
+
+
+def test_negative_run_capsule_upgrade_playbook_on_satellite(target_sat):
+    """Run Capsule Upgrade playbook against the Satellite itself
+
+    :id: 99462a11-5133-415d-ba64-4354da539a34
+
+    :steps:
+        1. Add REX key to the Satellite server.
+        2. Run the Capsule Upgrade Playbook.
+        3. Check the job output for proper failure reason.
+
+    :expectedresults: Should fail
+
+    :CaseImportance: Medium
+    """
+    sat = target_sat.nailgun_host
+    template_id = (
+        target_sat.api.JobTemplate()
+        .search(query={'search': 'name="Capsule Upgrade Playbook"'})[0]
+        .id
+    )
+
+    target_sat.add_rex_key(satellite=target_sat)
+    with pytest.raises(TaskFailedError) as error:
+        target_sat.api.JobInvocation().run(
+            data={
+                'job_template_id': template_id,
+                'inputs': {
+                    'target_version': CAPSULE_TARGET_VERSION,
+                    'whitelist_options': "repositories-validqqate,repositories-setup",
+                },
+                'targeting_type': "static_query",
+                'search_query': f"name = {sat.name}",
+            }
+        )
+    assert 'A sub task failed' in error.value.args[0]
+    job = target_sat.api.JobInvocation().search(
+        query={'search': f'host={sat.name},status=failed,description="Capsule Upgrade Playbook"'}
+    )[0]
+    response = client.get(
+        f'{target_sat.url}/api/job_invocations/{job.id}/hosts/{sat.id}',
+        auth=(target_sat.username, target_sat.password),
+        verify=False,
+    )
+    assert 'This playbook cannot be executed on a Satellite server.' in response.text
+
+
+@pytest.mark.rhel_ver_list([7])
+def test_positive_use_alternate_directory(rex_contenthost, module_org, target_sat):
```

Comment:
module_org can be dropped here, @omkarkhatavkar, could you do me a favor and remove it? I'm closing this pr https://github.com/SatelliteQE/robottelo/pull/9699 that was making the same change in the old location in favor of this pr